### ISSUE TITLE
Theme not applied to terminal

### ISSUE THREAD

**Issue by Alex-vZyl** [neovide/neovide#1528]
**Describe the bug**

Neovide does not apply the theme's terminal colors to the terminal instance.  I am using [gruvbox-material](https://github.com/sainnhe/gruvbox-material) and AFAICT they have terminal colors specified.

**To Reproduce**

1. Apply theme.
2. Open terminal.

**Expected behavior**

The theme should be applied to the terminal instance.

**Screenshots**

Neovide is on the left, with Neovim on the right.

![image](https://user-images.githubusercontent.com/81622310/188725449-c062401b-2820-4921-acf2-86fef946f071.png)

**Desktop (please complete the following information):**
 - Arch Linux
 - Neovide Version 0.10.1
 - Neovim Version 0.7.2

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**

The file content is too long.  if it is required I will upload it somewhere.

**Additional context**

None.

**Comment by MultisampledNight** [comment]
Thanks for reporting! I can't seem to reproduce the issue with this minimal config utilizing vim-plug though:

```vim
call plug#begin()
Plug 'sainnhe/gruvbox-material'
call plug#end()
```

Running `:PlugInstall`, `:colo gruvbox-material` and finally `:term`, I do see terminal colors being applied correctly:

![Neovide with a split, left a terminal showing neofetch, right an empty buffer, all in gruvbox-material.](https://user-images.githubusercontent.com/80128916/188733503-44ce9361-1640-44a6-9525-16b8e67f3e7e.png)


Looking [at the git blame](https://github.com/sainnhe/gruvbox-material/blame/1df9af2ebe8c6e0204fb00318c0febe76786b4dc/colors/gruvbox-material.vim#L477-L492), the last time the relevant lines were changed was 2 years ago. Perhaps you didn't update your plugins for quite a time?

Related: #1398.

**Comment by Alex-vZyl** [comment]
Okay so I tested everything with a minimal `init.vim` file like you showed and everything worked fine.  I then went back to my original `init.vim` file and now everything is working again...  I have no idea what happened.

**Comment by Alex-vZyl** [comment]
With a minimal `init.vim` it is also working on my side, but I found the culprit.  I also tested it with the minimal file and it removes the theme from the terminal.
```vim
let g:gruvbox_material_disable_terminal_colors = 1
```
This line is supposed to stop the theme from being applied to the terminal so that the default terminal theme (outside of neovim) can be applied.  From their help page:
```
------------------------------------------------------------------------------
                                    *g:gruvbox_material_disable_terminal_colors*
g:gruvbox_material_disable_terminal_colors 

Setting this option to `1` will disable terminal colors provided by this color
scheme so you can remain terminal colors the same when using |:terminal| in
vim and outside of vim.

    Type:               |Number|
    Available values:   `0`, `1`
    Default value:      `0`

------------------------------------------------------------------------------
```

So expected behaviour is to just have my normal fish instance colors.  This is what happens in neovim inside alacritty.

I am unsure what causes this and if this is a Neovide issue.

**Comment by MultisampledNight** [comment]
Yeah, I suggest you to remove this flag. There's no sane way Neovide could find out about

1. which terminal emulator you use
2. get its colors from it (cross-terminal, while also launching it properly)

If you'd still prefer to have it active in a real terminal emulator should you ever change your colorscheme for the terminal emulator but want to keep your Vim colorscheme, you might want to set it conditionally depending on whether `g:neovide` exists or not.

**Comment by Alex-vZyl** [comment]
Fair enough.  This is actually expected behavior then.

Maybe add an entry in the FAQ about this?  Other than that I think this issue is resolved.

**Comment by MultisampledNight** [comment]
Added in 6c0c1828f45dba6c44b0fa1004ca0c67afdb8cbb.
